frontend/packages/ceph-storage-plugin/console-extensions.json

```
@@ -54,7 +54,7 @@
   {
     "type": "console.storage-provider",
     "properties": {
-      "name": "ObjectBucketClaim",
+      "nameKey": "%ceph-storage-plugin~ObjectBucketClaim%",
```

I think this can be

```suggestion
      "name": "%ceph-storage-plugin~ObjectBucketClaim%",
```

with no other changes. It looks like `translateExtension` will automatically replace the string without needing to call `t` on the value later.

https://github.com/spadgett/console/blob/f34c7465269643a614fc622daf0856f73c6fb785/frontend/packages/console-plugin-sdk/src/utils/extension-i18n.ts#L14-L24